js/main.js

```diff
@@ -45,7 +33,50 @@ if (shouldQuit) {
 app.on('ready', getUrlAndOpenMainWindow);
 
 function getUrlAndOpenMainWindow() {
-    // for dev env allow passing url argument
+    let installMode = false;
+    
+    process.argv.some((val) => {
+        let flag = '--install';
+        if (val === flag) {
+            installMode = true;
+            getConfigField('launchOnStartup')
+            .then(setStartup);
+        }
+
+        return false;
+    });
+
+    if (installMode === false){
+        openMainWindow();
+    }
+}
+
+function setStartup(lStartup){
+    if (lStartup === true){
+        symphonyAutoLauncher.isEnabled()
```

Comment:
why can't this function be simplified to:
function setStartup(lStartup){
  if (lStartup) {
     return symphonyAutoLauncher.enable();
  } else {
   return symphonyAutoLauncher.disable();
 }
}